client/web/src/jscontext.ts

```diff
@@ -41,6 +41,8 @@ export interface SourcegraphContext extends Pick<Required<SiteConfiguration>, 'e
 
     githubAppCloudSlug: string
 
+    githubAppCloudClientID: string
```

Comment:
```suggestion
    githubAppCloudClientID: string
```

nit: do we want to remove this empty line, since every other place seems to putting these two variables together?

Reply:
I kept the gap there because the rest of the file has a gap between each line item, regardless of their groupability